styles/base.css

```diff
@@ -0,0 +1,30 @@
+body {
+  background-color: #f5f6f7;
+  font-family: "Lora","open sans","HelveticaNeue-Light",
+                "Helvetica Neue Light","Helvetica Neue",
+                "Arial","Lucida Grande","sans-serif";
+color: #63707e;
+}
+
+.profilePic {
+  max-width: 35%;
+}
+
+.profilePic  {
+  width: 75%;
+  display: block;
+  margin: 10px auto;
+}
+
+img {
+  max-width: 55%
+}
+
+img {
+  width: 100%;
```

Comment:
this width is overriding the max-width above in desktop view (that's why your images are huge)
so you may want to add a max-width to a media query